pkg/jx/cmd/import.go

```diff
@@ -940,6 +940,7 @@ func (options *ImportOptions) ensureDockerRepositoryExists() error {
 		return err
 	}
 
+	region, _ := amazon.ReadRegion(kubeClient, ns)
```

Comment:
This should be cloud provider independent. It should be fine after moving the function in `kube` package.

Reply:
Done.